site/src/components/sections/Sections.vue

```diff
@@ -91,18 +91,21 @@
           <span
             class="padding-left prerequisiteError"
             :class="{
-              hidden: section.rem <= 0 || !section.closed,
+              hidden:
+                section.xl_rem === undefined ||
+                section.xl_rem > 0 ||
+                section.rem <= 0,
```

Comment:
This logic looks right but at first glance it's a bit fishy - can you add a comment explaining that the warning is only shown when there are seats available in a section but there are not cross-listed seats available?

Reply:
It avoids displaying "full section" and "full course (no XL seats") at the same time. I'm happy to swap once one is shown but I dont think we should show both.

Reply:
Oh no I agree that the messages should be different, my point was that there should be a comment for that boolean check explaining what it does a bit more verbosely.

Reply:
Ah I missed the comment part - yeah I agree, comment is necessary.

Reply:
@benjaminrsherman Resolved in latest push